firmware/ci.c

```diff
@@ -27,6 +27,18 @@
 
 int status_enabled;
 
+static const struct {
```

Comment:
It makes more sense in the edid file. It is only used in that file, but it would be cleaner in the file it is relevant to.